.vim/common_config/01_plugin_config.vim

```diff
@@ -231,6 +231,10 @@ call plug#begin('~/.vim/plugged')
   let g:syntastic_javascript_checkers = ['eslint']
   let g:syntastic_scss_checkers = ['stylelint']
 
+" Autoformat to follow conventions
+Plug 'Chiel92/vim-autoformat'
+" Perform Autoformat on save, comment this to disable
+au BufWrite * :Autoformat
```

Comment:
I think its better to make a keymap to autoformat instead of automatically running autoformat when saving file.